conda/Dockerfile

```diff
@@ -41,8 +41,12 @@ RUN bash ./install_conda.sh && rm install_conda.sh
 
 # Install CUDA
 FROM base as cuda
+ARG CUDA_VERSION=10.2
```

Comment:
Why is it 10.2 CUDA ? We pretty much deprecated it everywhere already, should we use something newer ? Like 11.6 ?